dev/io.openliberty.microprofile.metrics.5.0.monitor.internal/src/io/openliberty/microprofile/metrics/internal/monitor/MonitorMetricsHandler.java

```diff
@@ -200,18 +218,66 @@ protected void register(String objectName, String[][] data) {
             metrics = new MonitorMetrics(objectName);
             metrics.createMetrics(sharedMetricRegistry, data);
             metricsSet.add(metrics);
+            if (isBetaModeCheck()) {
+                // Register vendor computed metrics
+                registerComputedMetrics(objectName, metrics);
+            }
             Tr.debug(tc, "Monitoring MXBean " + objectName + " is registered to mpMetrics.");
         } else {
             Tr.debug(tc, objectName + " is already registered.");
         }
 
     }
 
+    protected void registerComputedMetrics(String objectName, MonitorMetrics monMetrics) {
+        if (objectName.contains("ServletStats") || objectName.contains("ConnectionPoolStats")) {
+            Set<MetricID> metricIDSet = monMetrics.getMetricIDSet();
+            Tr.debug(tc, "Registering computed metric for " + metricIDSet);
+            cmmh.createComputedMetrics(objectName, metricIDSet);
+        }
+    }
+
+    protected void unregisterComputedMetrics(String objectName, MonitorMetrics monMetrics) {
+        if (objectName.contains("ServletStats") || objectName.contains("ConnectionPoolStats")) {
+            Set<MetricID> metricIDSet = monMetrics.getMetricIDSet();
```

Comment:
debug statement to match the registering method's one. Also helpful for debugging if need be.

Reply:
Done.